test/unit/spec/api/upaApiSpec.js

```diff
@@ -157,6 +162,7 @@ define(['api/upa', 'narrativeConfig'], (UpaApi, Config) => {
         });
 
         it('Should fail if the workspace id cannot be found.', () => {
+            const prevWsId = Config.get('workspaceId');
```

Comment:
Not storing and resetting the globally known `workspaceId` was occasionally making failures. This really shouldn't muck with the global space anyway, but this is a fix.